components/authz-service/storage/postgres/migration/legacy/iam.go

```diff
@@ -501,6 +512,11 @@ func convertV1Resource(resource string) (string, error) {
 }
 
 func convertV1Cfgmgmt(terms []string) (string, error) {
+	if len(terms) == 1 {
+		// skip?
+		return "", nil
+	}
+
```

Comment:
This line in convertV1Actions looks like it might need a length guard as well.

https://github.com/chef/automate/pull/3351/files#diff-586ab1c0cc234c002424a036a86fbf85R593

Reply:
We won't hit this case any longer because I moved [the `convertResource` function ahead of the `convertAction`](https://github.com/chef/automate/pull/3351/files#diff-586ab1c0cc234c002424a036a86fbf85L388). So if there's a single resource, like `ingest`, the `convertResource `function returns "" and then the function exits early, skipping the policy.